Prototyper/Classes/Feedback/EditScreenshotView.swift

```diff
@@ -97,17 +97,21 @@ struct EditScreenshotView: View {
     }
     
     private func save() {
-        self.drawings = [Drawing]()
-        self.model.screenshot = UIApplication.shared.windows.first?.asImage(rect: rect) ?? UIImage()
+        self.model.markupDrawings.removeAll()
+        self.allDrawings.forEach({ self.model.markupDrawings.append($0) })
+        self.model.screenshotWithMarkup = UIApplication.shared.windows.first?.asImage(rect: rect) ?? UIImage()
         self.presentationMode.wrappedValue.dismiss()
     }
     
     private func cancel() {
         if colorPickerShown {
             colorPickerShown = false
         } else {
-            self.drawings = [Drawing]()
             self.presentationMode.wrappedValue.dismiss()
         }
     }
+    
+    private func setupCurrentDrawings() {
+        self.model.markupDrawings.forEach({ self.allDrawings.append($0) })
```

Comment:
As far as I can see there is no need to iterate over the `markupDrawings`
```suggestion
        self.allDrawings = self.model.markupDrawings
```